src/mocks/msw/handlers/entityHandlers.ts

```diff
@@ -51,12 +45,13 @@ export const entityHandlers = [
       }
       if (req.body) {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        const requestBody = req.body as any
-        if (requestBody.name === MOCK_FILE_NAME) {
-          response = mockFileEntity
-          status = 200
-        } else if (requestBody.name === MOCK_PROJECT_NAME) {
-          response = mockProjectEntity
+        const requestBody = req.body as Entity
+
+        const entityData = mockEntities.find(
+          entity => entity.name === requestBody.name,
+        )
```

Comment:
In a given handler, find the entity that matches by ID, and return the requested data.